pkg/sloop/store/untyped/store.go

```diff
@@ -94,6 +95,8 @@ func OpenStore(factory badgerwrap.Factory, config *Config) (badgerwrap.DB, error
 		opts = opts.WithValueLogLoadingMode(options.FileIO)
 	}
 
+	opts = opts.WithTruncate(config.BadgerVLogTruncate)
```

Comment:
one nit pick, I see other examples in this function give a if condition check before set opts, one example is: https://github.com/salesforce/sloop/blob/master/pkg/sloop/store/untyped/store.go#L93, maybe we should follow the same practice for code consistence.

Reply:
its a boolean value -> if clause is unnecessary op

Reply:
And regarding the consistency, line 100 below does the same, just assigns the variable as its a boolean